drivers/led/lp50xx.c

```diff
@@ -193,6 +194,18 @@ static int lp50xx_init(const struct device *dev)
 		return -EINVAL;
 	}
 
+	/* Enable EN pin if given in devicetree. */
+	if (config->en_pin.port != NULL) {
+		if (device_is_ready(config->en_pin.port)) {
+			gpio_pin_configure_dt(&config->en_pin, GPIO_OUTPUT_HIGH);
```

Comment:
I think it should be GPIO_OUTPUT_ACTIVE instead. You should not assume how polarity is configured in DT.